modules/codegen/src/main/scala/com/twilio/guardrail/generators/Http4sServerGenerator.scala

```diff
@@ -246,7 +246,7 @@ object Http4sServerGenerator {
                 None,
                 Some(
                   (
-                    q"urlForm.values.get(${arg.argName.toLit}).map(_.map(Json.fromString(_).as[$tpe]).sequence).sequence",
+                    q"urlForm.values.get(${arg.argName.toLit}).flatMap(_.toList.headOption).map(Json.fromString(_).as[$tpe]).sequence.sequence",
```

Comment:
Doesn't the `headOption` defeat the purpose of `multiOpt` here? Maybe I'm missing something?

If you resolve the formatting issues (which are done first to prevent unnecessarily long builds that end with formatting failure at the very end) it would be interesting to see if all the regression tests still pass.

Reply:
Should be fixed